src/content/api/cli.md

```diff
@@ -70,49 +142,90 @@ If your project structure is as follows -
 ```
 
 ```bash
-webpack src/index.js -o dist/bundle.js
+webpack ./src/index.js -o dist
 ```
 
-This will bundle your source code with entry as `index.js`, and the output bundle file will have a path of `dist`, and the filename will be `bundle.js`
+This will bundle your source code with entry as `index.js`, and the output bundle file will have a path of `dist`.
 
 ```bash
-	| Asset     | Size    | Chunks      | Chunk Names |
-	|-----------|---------|-------------|-------------|
-	| bundle.js | 1.54 kB | 0 [emitted] | index       |
-	[0] ./src/index.js 51 bytes {0} [built]
-	[1] ./src/others.js 29 bytes {0} [built]
+Hash: 07a1c2d056e28ff1cab2
+Version: webpack 4.44.2
+Time: 174ms
+Built at: 10/10/2020
+  Asset       Size  Chunks             Chunk Names
+main.js  952 bytes       0  [emitted]  main
+Entrypoint main = main.js
+[0] ./src/index.js 30 bytes {0} [built]
+[1] ./src/others.js 0 bytes {0} [built]
 ```
 
 ```bash
-webpack ./src/index.js ./src.index2.js -o dist/bundle.js
+webpack ./src/index.js ./src/others2.js -o dist/
 ```
 
 This will form the bundle with both the files as separate entry points.
 
 ```bash
-	| Asset     | Size    | Chunks        | Chunk Names   |
-	|-----------|---------|---------------|---------------|
-	| bundle.js | 1.55 kB | 0,1 [emitted] | index, index2 |
-	[0] ./src/index.js 51 bytes {0} [built]
-	[0] ./src/index2.js 54 bytes {1} [built]
-	[1] ./src/others.js 29 bytes {0} {1} [built]
+Hash: fad168056241c7181505
+Version: webpack 4.44.2
+Time: 175ms
+Built at: 10/10/2020
+  Asset        Size  Chunks             Chunk Names
+main.js  1010 bytes       0  [emitted]  main
+Entrypoint main = main.js
+[0] multi ./src/index.js ./src/others2.js 40 bytes {0} [built]
+[1] ./src/index.js 30 bytes {0} [built]
+[2] ./src/others.js 0 bytes {0} [built]
+[3] ./src/others2.js 0 bytes {0} [built]
 ```
 
+## Default Configurations
+
+CLI will look for some default configurations in the path of your project, here are the config files picked up by CLI.
+
+If no `mode` is supplied via flags or config then this is the lookup order in increasing order
 
-### Common Options
+> example - config file lookup will be in order of .webpack/webpack.config.development.js > webpack.config.development.js > webpack.config.js
+
+```md
+'webpack.config',
+'webpack.config.dev',
+'webpack.config.development',
+'webpack.config.prod',
+'webpack.config.production',
```

Comment:
If no `--mode` is specified this is a bit weird. No mode defaults to production, so it should also default to choosing the production config.

Reply:
Ah right mode defaults to production, this needs a little update on the CLI side to default to production config, will take care of it.